openc3/python/openc3/packets/structure.py

```diff
@@ -481,9 +481,24 @@ def buffer(self, buffer):
     def clone(self):
         struct = copy.copy(self)
         struct._buffer = self.buffer  # Makes a copy
+        struct.accessor = copy.copy(self.accessor)
         struct.accessor.packet = struct
         return struct
 
+    # Clone that also deep copies items
+    # @return [Structure] A deep copy of the structure
+    def deep_copy(self):
```

Comment:
Python already has a `deepcopy` method: https://docs.python.org/3/library/copy.html

Reply:
Its a mostly deep copy.  Can't do a real deep copy at least in Ruby because of the Mutex